dluhc-component-library/src/components/timetablePage/AccordionDropdown.tsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import { IndicationArrow } from "../inlinesvgs/IndicationArrow";
+
+const AccordionDropdown = ({ information }: { information?: string }) => {
+  const [isClicked, setClicked] = useState(false);
+
+  const toggle = () => {
+    isClicked == true ? setClicked(false) : setClicked(true);
+  };
+
+  return (
+    <>
+      <div className="pt-1 pb-3">
+        <IndicationArrow
+          className={
+            "h-3 w-3 transition-all inline-block" +
+            (isClicked ? " rotate-90" : "")
+          }
+        />
+        <span
+          className="text-blue-400 underline cursor-pointer pl-2"
+          onClick={() => toggle()}
```

Comment:
When we have a function that is called as part of an `on` prop we should call it `handleX` and be verbose for what its handling so in this case `handleDropdownClicked`. It could be fine to then call a "toggle" function inside there if we needed to be we can just rename out existing one here.

We also dont need the `() =>` here so this line should just be: `onClick={handleDropdownClicked}`